cmd/bootstrap/main.go

```diff
@@ -98,6 +99,15 @@ func main() {
 	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefixHttp, AltPrefix: envPrefix}); err != nil {
 		logger.Fatal(fmt.Sprintf("failed to load %s HTTP server configuration : %s", svcName, err))
 	}
+	tp, err := jaegerClient.NewProvider(svcName, cfg.JaegerURL)
```

Comment:
Since you are not using `traceprovider` no need to initialize it

Reply:
If not initialized we don't get traces, so I'll leave it for testing purposes and rebase when the other PR is merged